solid-acp.ttl

```diff
@@ -0,0 +1,281 @@
+acp:resource
+    a rdf:Property ;
+    owl:inverseOf acp:accessControlResource ;
+    rdfs:label "resource"@en ;
+    dc:description "The resource property connects ACRs to resources they control. It is the inverse of acp:accessControlResource."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource .
+
+acp:accessControlResource
+    a rdf:Property ;
+    owl:inverseOf acp:resource ;
+    rdfs:label "access control resource"@en ;
+    dc:description "The access control resource property connects resources to ACRs controlling access to them. It is the inverse of acp:resource."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:range acp:AccessControlResource .
+
+acp:accessControl
+    a rdf:Property ;
+    rdfs:label "access control"@en ;
+    dc:description "The access control property connects ACRs to Access Controls."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource ;
+    rdfs:range acp:AccessControl .
+
+acp:memberAccessControl
+    rdfs:label "member access control"@en ;
+    dc:description "The member access control property transitively connects ACRs of member resources to Access Controls."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControlResource ;
+    rdfs:range acp:AccessControl .
+
+acp:apply
+    a rdf:Property ;
+    rdfs:label "apply"@en ;
+    dc:description "The apply property connects Access Controls to the Policies they apply to resources."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:AccessControl ;
+    rdfs:range acp:Policy .
+
+acp:allow
+    a rdf:Property ;
+    rdfs:label "allow"@en ;
+    dc:description "The allow property connects Policies to the Access Modes they allow if satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:AccessMode .
+
+acp:deny
+    a rdf:Property ;
+    rdfs:label "deny"@en ;
+    dc:description "The deny property connects Policies to the Access Modes they deny if satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:AccessMode .
+
+acp:allOf
+    a rdf:Property ;
+    rdfs:label "all of"@en ;
+    dc:description "The allOf property connects Policies to a set of Matchers, all of which MUST be satisfied for the Policy to be satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:Matcher .
+
+acp:anyOf
+    a rdf:Property ;
+    rdfs:label "any of"@en ;
+    dc:description "The anyOf property connects Policies to a set of Matchers, at least one of which MUST be satisfied for the Policy to be satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:Matcher .
+
+acp:noneOf
+    a rdf:Property ;
+    rdfs:label "none of"@en ;
+    dc:description "The noneOf property connects Policies to a set of Matchers, all of which MUST NOT be satisfied for the Policy to be satisfied."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:domain acp:Policy ;
+    rdfs:range acp:Matcher .
```

Comment:
```suggestion
acp:allOf
    a rdf:Property ;
    rdfs:label "all of"@en ;
    dc:description "The all of property connects Policies to a set of Matchers, all of which MUST be satisfied for the Policy to be satisfied."@en ;
    rdfs:isDefinedBy acp: ;
    rdfs:domain acp:Policy ;
    rdfs:range acp:Matcher .

acp:anyOf
    a rdf:Property ;
    rdfs:label "any of"@en ;
    dc:description "The any of property connects Policies to a set of Matchers, at least one of which MUST be satisfied for the Policy to be satisfied."@en ;
    rdfs:isDefinedBy acp: ;
    rdfs:domain acp:Policy ;
    rdfs:range acp:Matcher .

acp:noneOf
    a rdf:Property ;
    rdfs:label "none of"@en ;
    dc:description "The none of property connects Policies to a set of Matchers, all of which MUST NOT be satisfied for the Policy to be satisfied."@en ;
    rdfs:isDefinedBy acp: ;
    rdfs:domain acp:Policy ;
    rdfs:range acp:Matcher .
```

The spelling of the property as per its label was intentional, I think it reads better and I don't think using the URI fragment instead improves meaningfulness.